### PR TITLE
task/4105 cms footer plugin

### PR DESCRIPTION
**Pull request by tiwariav**
A new plugin created for footer in ungleich page. Contents that can be edited are:
- Copyright name
- url to 'twitter', 'linkedin' and 'github' profiles (more can be added in future, if required)
- text and url for the link on the right side of the footer.

### CODE REVIEW

**Comment by tiwariav** [comment]
@pcoder to use this, we'll have to remove the existing footer from the base template  `ungleich_cms_page.html`. So should we create a new base file, or remove it from the existing one and add footers to all other pages using the existing base template?

File: ungleich_page/models.py [hunk 1]
```diff
@@ -222,3 +222,24 @@ class UngleichHTMLOnly(CMSPlugin):
 
     def __str__(self):
         return self.name
+
+
+class UngleichFooter(CMSPlugin):
+    copyright = models.CharField(max_length=100, default='', blank=True)
```
Comment (pcoder) [inline ungleich_page/models.py]: Probably add a help line explaining that this is name of the company alongside the copyright year

**Comment by verysanghee** [comment]
So from what i test, this plugin update affects all current pages under CMS, and this is opposite of what we want to achieve. We need a footer plugin which affects only the page the plugin is installed and leaves the other pages as they are. 
ex) when we update the footer of Digital Glarus to redirect to twitter account of Digital Glarus, the ungleich.ch twitter shoulnd't be affected by it and should only keep current ungleich twitter link.

**Comment by verysanghee** [comment]
where are we with this PR? :) Is it now good to go? @tiwariav

**Comment by tiwariav** [comment]
Yes its good @verysanghee . Just waiting for a review.

File: ungleich_page/templates/ungleich_page/ungleich/_footer.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+			      	<a href="{{instance.youtube_url}}"><i class="fa fa-youtube-play"></i></a>
+			      </li>
+		    	{% endif %}
+		    </ul>
+		  </div>
+		  <div class="col-md-4">
+		  	{% if instance.link_text %}
+			    <ul class="list-inline quicklinks">
+			      <li>
+			      	<a href="instance.link_url">{{instance.link_text}}</a>
```
Comment (pcoder) [inline ungleich_page/templates/ungleich_page/ungleich/_footer.html]: Missing {{ }} here ...

**Comment by pcoder** [comment]
Looks good to me ..

**Pull request merged**